pandas/src/generated.pyx

```diff
@@ -9110,6 +9110,8 @@ def group_count_bin_float64(ndarray[float64_t, ndim=2] out,
         ndarray[int64_t, ndim=2] nobs = np.zeros((out.shape[0], out.shape[1]),
                                                  dtype=np.int64)
 
+    if len(bins) == 0:
```

Comment:
this is by definition **generated** code, edits here will be lost. rather make the fix in ``src/generate_code.py``, then``python generate_code.py`` creates the ``generated.pyx``